docs/user_guide/notes_admin.rst

```diff
@@ -37,3 +37,15 @@ Users may receive 500 Internal Server Error in some extreme cases, for example:
 * Database or storage path has incorrect permission, or disk is full.
 * There are too many files (probably more than :math:`10^{18}`) created and
   causes Version 4 UUID collision in ``json_files_unpack()``.
+
+Limitations
+-----------
+* ngshare cannot run concurrently, which may be a bottle neck if too many users
+  are using this service.
+* ngshare stores all uploaded files in a same directory. This may create
+  performance issues when there are too many files uploaded.
+* Currently there are no control on user uploads (e.g. file size, number of
+  files).
+* Admin user names cannot contain "," (comma sign).
+* User names are not designed to be changable.
```

Comment:
What exactly do you mean here? "not designed to be changeable"?

Reply:
If one user named `jkl` submitted sth, and this user is removed by admin, and another user is created with name `jkl`, then the new user can see what the original user submitted.

Reply:
ok, I see to me this made it sound like you could not change a username, which I believe we can. Maybe we can rephrase it or elaborate.